src/scripts/notebooks/clinvar-udn-submission-analysis/data/output/clinvar_udn_submissions-no_mondo-disease_name_only.txt

```diff
@@ -0,0 +1,145 @@
+Name
+1p13.3 deletion syndrome
+ABCA2-related disorder
+ACOX1-related disorder
+ADCY2-related disorder
+ADGRV1-related myoclonic epilepsy
+AGTPBP1-related disorder
+AIFM1-related hypomyelination with spondylometaphyseal dysplasia
+ARX-associated condition
+ATAD3 gene cluster related condition
+ATP5F1A-related disorder
+ATP5PO-related disorder
+AXIN2-related disorder
+Adult hypophosphatasia
+BAP1-associated neurodevelopmental disorder
+BICRA-related disorder
+BLK-related disorder
+Bone mineral density quantitative trait locus 18
+CACNA1A-related disorder
+CACNA1C-related disorder
+CDH2-related disorder
+CDK19-related disorder
+CDKL5-related disorder
+CHRNA3-related disorder
+COPB2-related disorder
+CRELD1-related disorder
+CTNNA1-associated FEVR
+Cerebellar atrophy

```

Comment:
this is in HPO, so should we also add it as a disease in Mondo?
http://purl.obolibrary.org/obo/HP_0001272

Reply:
The disease-phenotype issue will be significant for this work it seems!